build.gradle

```diff
@@ -60,7 +60,9 @@ test {
 }
 
 dependencies {
-    compile 'net.bytebuddy:byte-buddy:1.4.20'
+    compile 'net.bytebuddy:byte-buddy:1.4.26'
+    compile 'net.bytebuddy:byte-buddy-agent:1.4.26'
+    compile 'com.blogspot.mydailyjava:weak-lock-free:0.11'
```

Comment:
@raphw Can we make the dependency optional ?, or maybe shadow the jar or just the needed classes ?